Makefile.toml

```
@@ -108,8 +108,14 @@ mkdir -p ${BUILDSYS_BUILD_DIR}
 mkdir -p ${BUILDSYS_OUTPUT_DIR}
 mkdir -p ${BUILDSYS_PACKAGES_DIR}
 mkdir -p ${GO_MOD_CACHE}
+'''
+]
 
-for cmd in bash curl docker gunzip lz4 ; do
+[tasks.build-preflight-check]
```

```suggestion
[tasks.setup-build]
```
Might need other setup tasks later, not just checks, and this is more consistent with the other task name.